Applications/SimpleShot/test/mock_test.cpp

```diff
@@ -1,16 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
how about use `git filter-repo --path <path to the file or directory> --invert-paths` to remove this file?

Reply:
I wonder if I should do this, this is intended as the 24d53d3d3853b55af95379e5823d9389039ce739 is just a scaffolding commit buildable with a commit to test.

`mock_test.cpp` is just renamed to `simpleshot_utils.cpp` but it seems that the git doesn't catch this rename

Reply:
> mock_test.cpp is just renamed to simpleshot_utils.cpp in the next commit but it seems that the git doesn't catch this rename

Then its fine.